### PR TITLE
Add new icon, functionality to open extension in browser

### LINKED ISSUE #22: BUG: Cannot copy and paste configuration options
Observed: It is not possible to configure the extension via copy and paste of parameters because the popup closes when you tab out to another window to copy another configuration parameter and loses in progress changes
Expected: Popup should not close, or remember the in progress configuration changes
Environment: Firefox on PopOS 20.04 (Ubuntu 20.04 based)
Steps to reproduce:
1) Install extension on firefox
2) Copy one of the configuration parameters into the clipboard (e.g. lotus endpoint)
3) Click "Filecoin" icon
4) Click "Options"
5) Paste the configuration parameter in the clipboard into a text box on the configuration screen
6) Tab to another application (e.g. slack) to copy the next configuration parameter (e.g. JWT Wallet)
7) Tab back to browser, observe that popup is no longer visible
8) Click "Filecoin" icon
9) Click "Options", observe that the first configuration item pasted on step 5 is gone

### LINKED ISSUE #31: Upload fails in Firefox
Uploading a file to the extension fails in Firefox, but works correctly in Chrome.  Maybe be a side effect of #29.

[Here is a video showing how it fails in Firefox.](https://drive.google.com/file/d/1EMF0QwPAbDt-HY_mwgEEHll7sHPQ9uzY/view?usp=sharing)

[Here is a video showing the desired behavior.](https://drive.google.com/file/d/1c0raB6BwV5J7jWUZ9tJ34ltllkbGswIn/view?usp=sharing)

### PR DESCRIPTION
**Pull request by ionutiorgulescu**

### CODE REVIEW

File: src/popup/App/OpenExtensionInBrowserIndicator/OpenExtensionInBrowserIndicator.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import OpenInBrowserIndicator from 'src/popup/components/OpenInBrowserIndicator';
+
+function OpenExtensionInBrowserIndicator(props) {
+
+  return (
+      <OpenInBrowserIndicator
+          icon={
+              <svg width="16" height="16" viewBox="0 0 433 466" version="1.1" xmlns="http://www.w3.org/2000/svg">
```
Comment (brunolm) [inline src/popup/App/OpenExtensionInBrowserIndicator/OpenExtensionInBrowserIndicator.js]: I think it'd be better to have all icons in a single place. Today it's in the `IconButton` component https://github.com/filecoin-shipyard/browser-retrieval/blob/master/src/popup/components/IconButton/IconButton.js

By doing this you could potentially use a single component too (e.g. keep the other and replace button/icon with `IconButton`)

**Review by brunolm — COMMENTED** [review]
This only works on Firefox, it could be interesting to have it work on Chrome/Edge if the effort to do so is small.

File: src/popup/App/OpenExtensionInBrowserIndicator/OpenExtensionInBrowserIndicator.js (hunk 1 above)
Comment (ghost) [reply src/popup/App/OpenExtensionInBrowserIndicator/OpenExtensionInBrowserIndicator.js]: @ionutiorgulescu Can you see if you can get this to work in at least Chrome?  Edge would be ideal too.

Also, I agree about having all the icons in one place.

Comment (ionutiorgulescu) [reply src/popup/App/OpenExtensionInBrowserIndicator/OpenExtensionInBrowserIndicator.js]: Just tested, it works in Chrome and Edge too. I will move the icon to IconButton component then give back to code review. Thanks guys!

**Review by brunolm — APPROVED** [review]

**Review by aistoc — APPROVED** [review]

**Pull request merged**